btk/draw_blends.py

```diff
@@ -288,8 +282,12 @@ def render_blend(self, blend_catalog, cutout, band):
             Images of blend and isolated galaxies as `numpy.ndarray`.
 
         """
-        if not hasattr(cutout, "mean_sky_level"):
-            raise AttributeError("cutout needs a mean_sky_level as an attribute.")
+        if not hasattr(cutout, "survey"):
+            mean_sky_level = cutout.mean_sky_level
+        elif not hasattr(cutout, "mean_sky_level"):
+            mean_sky_level = cutout.survey.mean_sky_level[[b == band for b in cutout.survey.bands]]
+        else:
+            raise AttributeError("cutout needs a `survey`  as an attribute.")
```

Comment:
I think this is fine for now, ultimately we would like all cutouts to be standarized (I think this is reasonable) to the point that all of them will have the `mean_sky_level` attribute (or something along these lines to calculate a background)

Reply:
Yeah I was thinking about that too, but I was unsure about it because for the WLD the attribute comes from WLD. But for cosmos since the survey carries all the informatiion it seems weird to pull the attribute out. I think this is an issue for the Cutout refactoring.